src/compiler/commandLineParser.ts

```diff
@@ -1245,6 +1192,93 @@ namespace ts {
         }
     }
 
+    type ParsedTsconfig = { include?: string[], exclude?: string[], files?: string[], options: CompilerOptions };
+
+    /**
+     * This *just* extracts options/include/exclude/files out of a config file.
+     * It does *not* resolve the included files.
+     */
+    function parseConfig(
+            json: any,
+            host: ParseConfigHost,
+            basePath: string,
+            configFileName: string,
+            resolutionStack: Path[] = [],

```

Comment:
Put the parameter with default values last? Otherwise the default value will never be used as it can not be emitted.